source/slang/slang-lower-to-ir.cpp

```diff
@@ -5408,6 +5413,10 @@ struct DeclLoweringVisitor : DeclVisitor<DeclLoweringVisitor, LoweredValInfo>
 
     bool isPublicType(Type* type)
     {
+        // TODO(JS):
+        // Not clear how should handle HLSLExportModifier here. 
+        // In the HLSL spec 'export' is only applicable to functions. So for now we ignore.
+
```

Comment:
If a type can't be marked `export`, then it seems like there is nothing to be done here?